keras_fsl/dataframe/operators/to_k_shot_dataset.py

```diff
@@ -31,8 +30,9 @@ def __init__(
             k_shot (int): number of consecutive crops from the same class
             preprocessing (function): to be applied onto the image after opening
             label_column (str): either "label_one_hot" or "label" depending on the expected form of the network
-            cache (Union[str, pathlib.Path]): cache directory to be passed to tf.data.Dataset.cache: each dataset (one per label) will be
-                cached in Path(cache) / label. No cleaning is done, see https://www.tensorflow.org/api_docs/python/tf/data/Dataset#cache.
+            cache (Union[str, Path]): cache directory to be passed to tf.data.Dataset.cache.
+                Each dataset, one per label, will be cached in Path(cache) / label.
+                No cleaning is done, see https://www.tensorflow.org/api_docs/python/tf/data/Dataset#cache.
```

Comment:
pourquoi on dit "no cleaning is done" alors qu'on met aussi le kwarg `reset_cache` ?

Reply:
j'ai pas modifié le commentaire et je pense que c'est le comportement par défaut ?

Reply:
ça te semble bien ou tu as une proposition ? enlever le no cleaning is done ou préciser utiliser reset_cache ?

Reply:
on retire la phrase (Vu ensemble)